BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -303,6 +303,7 @@
   gPayloadTokenSpaceGuid.PcdCsmeUpdateEnabled             | $(ENABLE_CSME_UPDATE)
   gPlatformCommonLibTokenSpaceGuid.PcdEmmcHs400SupportEnabled | $(ENABLE_EMMC_HS400)
   gPlatformCommonLibTokenSpaceGuid.PcdPreOsCheckerEnabled | $(ENABLE_PRE_OS_CHECKER)
+  gPlatformCommonLibTokenSpaceGuid.PcdForceToInitSerialPort | FALSE

```

Comment:
this is not actually required since it is the same as defined in  DEC.

Reply:
Removed in next push.